opencraft/settings.py

```diff
@@ -331,6 +331,20 @@
     default={"ram": 8192, "disk": 80}
 )
 
+# Openstack fails to connect randomly, thus we retry on certain status codes.
+# for example Openstack throws 400, and it will work if we keep retrying,
+# more on https://github.com/open-craft/opencraft/pull/805
+OPENSTACK_RETRYABLE_STATUS_CODES = env.list(
+    'OPENSTACK_RETRYABLE_STATUS_CODES',
+    default=list(range(500, 600))
+)
+
+# List of HTTP methods that we will retry
```

Comment:
@shimulch Could you update this with `Set of uppercased HTTP method verbs that we should retry on.` 

This is taken from the `urllib3` documentation. They use [uppercase HTTP methods](https://github.com/urllib3/urllib3/blob/1.26.5/src/urllib3/util/retry.py#L446) for comparison and I think it would be good to mention that information here.

Reply:
@arjunsinghy96, Thanks for the suggestion. I've addressed your review comment.